week_1/String/longest_common_prefix.js

```diff
@@ -0,0 +1,21 @@
+// Write a function to find the longest common prefix string amongst an array of strings.
```

Comment:
Looks like this is a duplicate file now?

Reply:
I deleted it